essence-of-live-coding/test/Util.hs

```diff
@@ -81,3 +81,18 @@ instance (Eq b, Show b) => Testable (CellSimulation a b) where
     , output1 = output
     , output2 = []
     }
+
+-- | Basic unit test for 'Cell' identities.
+--   Check whether one cell behaves the same as another cell.
+data CellIdentitySimulation a b = CellIdentitySimulation
+  { cella :: Cell Identity a b
+  , cellb :: Cell Identity a b
+  , commonInput :: [a]  
```

Comment:
```suggestion
  { cell1 :: Cell Identity a b
  , cell2 :: Cell Identity a b
  , input :: [a]  
```
So it's more like the other types, and there is no confusion with `a`/`b`

Reply:
I was actually trying to avoid the names used in the other CellMigrationSimulation, unfortunately this is the dreaded issue of these functions ending up in a common namespace. Is there way around the record issue ?

Reply:
Ah ok I see. I typically don't mind because I rarely use the record functions. I just like record syntax & wildcards, and there it's no trouble if the names are duplicate.

Reply:
We can't use the same names, so I changed them to cell1', cell2' and input'.